types/office-js-preview/index.d.ts

```diff
@@ -7247,15 +7247,23 @@ declare namespace Office {
          *
          * @remarks
          *
-         * **Important**: The `isDarkTheme` property isn't yet supported in Outlook. 
+         * **Important**: The `isDarkTheme` property isn't supported in Outlook. To determine the current theme in Outlook,
+         * use the body and control color properties instead. To automatically detect theme changes, use the
```

Comment:
could something like this work to clarify about the property names? - here and other instances

```suggestion
         * use the `body*Color` and `control*Color` properties instead. To automatically detect theme changes, use the
```